src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -1214,7 +1220,7 @@ void processZoneEquipSplitterInput(EnergyPlusData &state,
         GetOnlySingleNode(state,
                           ip->getAlphaFieldValue(objectFields, objectSchemaProps, "zone_equipment_outlet_node_name"),
                           state.dataZoneEquip->GetZoneEquipmentDataErrorsFound,
-                          DataLoopNode::ConnectionObjectType::SpaceHVACZoneEquipmentSplitter,
+                          thisZeqSplitter.spaceEquipType,
```

Comment:
Hmm.  OK.  So you previously sent in one of the enums directly, hardwired in as a splitter enum value.  Now instead you assign that enum to a splitter object member variable and pass that member variable in.  I'm not saying this is wrong, I just can't immediately tell the reason.  Maybe you could just clarify a bit, or wave your hand and say it has to be done for spaces :rofl: :rofl: